composables/helpers/buildSearchOrdersVars.ts

```diff
@@ -0,0 +1,17 @@
+export const buildSearchOrdersVars = (params: {
```

Comment:
Please rename " buildSearchOrdersVars" const and file name to "buildOrdersFilterInput" or "buildOrdersFilter".
We should have consistency on file names.
Note: we have another file name as "buildProductSearchVars", which we should also correct it.

Reply:
Resolved